### PR TITLE
Add daily_fil stats table

### PR DESCRIPTION
**Pull request by PatrickNercessian**

### CODE REVIEW

File: migrations/008.do.daily-fil-stats.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE daily_fil (
+    day DATE NOT NULL,
+    to_address TEXT NOT NULL,
+    amount BIGINT NOT NULL,
+    PRIMARY KEY (day, to_address)
+)
```
Comment (juliangruber) [inline migrations/008.do.daily-fil-stats.sql:6]:
```suggestion
)

```

File: migrations/008.do.daily-fil-stats.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE daily_fil (
```
Comment (juliangruber) [inline migrations/008.do.daily-fil-stats.sql:1]: To me, `daily_fil` isn't self explanatory. And what will this table track? Is it the FIL sent to addesses, or when we increased their balance in the smart contract?

**Review by juliangruber — CHANGES_REQUESTED** [review]

File: migrations/008.do.daily-fil-stats.sql (hunk 2 above)
Comment (bajtos) [reply migrations/008.do.daily-fil-stats.sql:1]: IIUC, we want to track rewards transferred every Monday from "scheduled rewards" to participant wallets. 

My proposals for a better table name:
- `daily_transfers` 
- `daily_reward_transfers`

We are using the `daily_` prefix in other table names to communicate that these tables store data with a daily granularity (i.e. they all use a `day DATE NOT NULL` column as the timestamp).

**Comment by PatrickNercessian** [comment]
Closed, as this is now owned by spark-stats, specifically in https://github.com/filecoin-station/spark-stats/pull/102

**Comment by PatrickNercessian** [comment]
Closed, as this is now owned by spark-stats, specifically in https://github.com/filecoin-station/spark-stats/pull/102

**Pull request closed without merging**